next/content/algorithms/sdr_notation.md

```diff
@@ -0,0 +1,1179 @@
+`$\ParentLabels_{\mathbf{u}_\total}: \Label^{[d_\total]} = [\Label_{u_\drg, l} \mid \forall u_\drg \in \mathbf{u}_\drg] \concat [\Label_{u_\exp, l - 1} \mid \forall u_\exp \in \mathbf{u}_\exp]$`\
+The arrays of a node `$v$`'s (where `$v$` is in Stacked-DRG layer `$l_v$`) DRG and total parent labels respectively (DRG parent labels are in layer `$l_v$`, expander parent labels are in layer `$l_v - 1$`). `$\ParentLabels_{\mathbf{u}_\drg}$` is only called while labeling nodes in layer `$l_0 = 0 \thin$`.
+
+`$\ParentLabels_{\mathbf{u}_\drg}^\star: \Label^{[N_\parentlabels]} = \ParentLabels_{\mathbf{u}_\drg}\dot\repeattolength(N_\parentlabels)$`\
+`$\ParentLabels_{\mathbf{u}_\total}^\star: \Label^{[N_\parentlabels]} = \ParentLabels_{\mathbf{u}_\total}\dot\repeattolength(N_\parentlabels)$`\
+The superscript `$^\star$` denotes that `$\ParentLabels$` has been expanded to length `$N_\parentlabels \thin$`.
+
+`$\R_{\porepchallenges, \batch}: \Byte^{[32]}$`\
+Randomness used to generate the challenge set for a batch of PoRep proofs.
+
+`$\PorepChallenges_{R, k}: (\NodeIndex \setminus 0)^{[N_{\porepchallenges / k}]}$`\
+The set of PoRep challenges for a replica `$R$`'s partition-`$k$` partition proof. The first node index `$0$` is not challenged in PoRep's (the operator `$\setminus$` is set subtraction).
+
+```text
+$\struct \PorepChallengeProof_c\ \{$
+$\quad \TreeDProof_c,$
+$\quad \ColumnProof_c,$
+$\quad \TreeRProof_c,$
+$\quad \ParentColumnProofs_{\mathbf{u}_\total}: {\ColumnProof_u}^{[d_\total]},$
+$\}$
+```
+The proof generated for each Merkle challenge `$c$` in a PoRep partition proof. The field `$\ParentColumnProofs_{\mathbf{u}_\total}$` stores the `$\ColumnProof_u$` for each parent `$u \in \mathbf{u}_\total$` of the challenge `$c$`.
+
+`$\PorepPartitionProof_{R, k}: \PorepChallengeProof^{[N_{\porepchallenges / k}]}$`\
+A single PoRep partition proof for partition `$k$` in a replica `$R$`'s PoRep proof batch.
+
+### PoSt
+
+`$\R_{\postchallenges, \batch \thin \aww}: \Fq$`\
+Randomness used to generate the challenge set for a batch of PoSt proofs.
+
+```text
+$\struct \PostReplica_\P\ \{$
+$\quad \TreeR,$
+$\quad \CommC,$
+$\quad \CommCR,$
+$\quad \SectorID,$
+$\}$
+```
+A replica `$R$` that a PoSt prover has access to. All fields are associated with `$R$` at the time of PoSt proof batch generation.
+
+`$\PostReplicas_{\P, \batch \thin \aww}: {\PostReplica}^{[*]}$`\
+The set of all distinct replicas that a PoSt prover `$\P$` has knowledge of at time of a Winning or Window batch proof generation. PoSt provers in the Filecoin network may have different sized replica sets, thus `$\PostReplicas$` is arbitrarily sized `$*$`.
+
+`$\PostReplicas_{\P, k \thin \aww}: {\PostReplica_{\P, \batch}}^{[0 < \ell \leq N_{\postreplicas/k \thin \aww}]}$`\
+`$\PostReplicas_{\P, k \thin \aww} =$`\
+`$\quad \PostReplicas_{\P, \batch \thin \aww}[k * N_{\postreplicas / k \thin \aww} \ldotdot (k + 1) * N_{\postreplicas / k \thin \aww}]$`\
```

Comment:
That looks really strange (and inconsistent). I've made all three lines into a block. Alternatively we could also put those two lines into a single line.

Let me know if you want to have it done differently than I did.

Reply:
Inconsistent because the first line would be highlighted differently than the last two?

I think that what you did in combining the three lines into one, looks good. I just didn't want the last two lines in separate code blocks, but I also wouldn't want the three lines highlighted differently.

Reply:
> Inconsistent because the first line would be highlighted differently than the last two?

Yes, as the font color is slightly different. They looked disconnected (compared to all other cases).

Sounds like the current style works, so I'll resolve this conversation.